src/App.vue

```diff
@@ -163,6 +163,27 @@
         />
     </div>
 
+
+
+    <div class="box">
+        <h3>Event CheckIn</h3>
+        <DatePicker
+            :i18n="{ night: 'Night',
+                      nights: 'Nights',
+                      'day-names': ['Mon', 'Tue', 'Wed', 'Thur', 'Fri', 'Sat', 'Sun'],
+                      'check-in': 'Check-in',
+                      'check-out': 'Check-Out',
+                      'month-names': ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+                    }"
+              v-on:check-in-changed="checkInchanged($event)"
```

Comment:
Can this be _checkInChanged_?

Reply:
done at commit 593b20cfd190852a96c946102f6923c9a301d026
thanks for youre review